src/npctalk.cpp

```diff
@@ -2955,6 +2976,8 @@ conditional_t::conditional_t( const std::string &type )
         set_is_driving( is_npc );
     } else if( type == "is_day" ) {
         set_is_day();
+    } else if( type == "player_has_stolen_item" ){
```

Comment:
```suggestion
    } else if( type == "u_has_stolen_item" ){
```
player functions are always `u_` and NPC functions are always `npc_`